### PR TITLE
Add dictionary default value

### PR DESCRIPTION
**Pull request by saschanaz**
Required after heycam/webidl#750. (Found from web-platform-tests/wpt#18382)

***
<a href="https://pr-preview.s3.amazonaws.com/saschanaz/pointerevents/pull/298.html" title="Last updated on Aug 23, 2019, 3:20 AM UTC (8ed941c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/pointerevents/298/ec5baa0...saschanaz:8ed941c.html" title="Last updated on Aug 23, 2019, 3:20 AM UTC (8ed941c)">Diff</a>

### CODE REVIEW

**Comment by saschanaz** [comment]
Marked as non substantive for IPR from ash-nazg.

File: index.html [hunk 1]
```diff
@@ -247,7 +247,7 @@ <h2><code>PointerEvent</code> Interface</h2>
     boolean     isPrimary = false;
 };
 
-[Constructor(DOMString type, optional PointerEventInit eventInitDict), Exposed=Window]
+[Constructor(DOMString type, optional PointerEventInit eventInitDict = {}), Exposed=Window]
```
Comment (NavidZ) [inline index.html:250]: I wonder what this means. Where is the default dictionary values that are specified before this line? Is there a way to set the default value to a dictionary with the values specified above? @saschanaz @Ms2ger

Comment (saschanaz) [reply index.html:250]: The `= {}` part should be same as passing JavaScript `{}`, which means those default values are automatically set.

Comment (patrickhlauke) [reply index.html:250]: i have to admit that i'm lost about the actual meaning of the whole IDL thing ... but I trust @saschanaz you know more what this is about etc (and it does look in line with what some other specs have done to update their IDL)

**Review by patrickhlauke — APPROVED** [review]

**Comment by patrickhlauke** [comment]
(going to leave it open for a bit just in case @ms2ger has some comment/suggestion about this)

**Review by Ms2ger — APPROVED** [review]
This is the correct change, yes. Thank you.

**Comment by patrickhlauke** [comment]
thanks for confirming @Ms2ger - and sorry for doubting you @saschanaz :+1:

**Pull request merged**